src/main/java/com/google/sps/data/ScheduledInterviewRequest.java

```diff
@@ -17,38 +17,38 @@
 /** Represents the data sent in a put or post request to the Person Servlet. */
 public class ScheduledInterviewRequest {
   private long id;
-  private TimeRange when;
-  private String interviewerId;
-  private String intervieweeId;
+  private String dateString;
+  private String interviewer;
+  private String interviewee;
+  private String role;
 
   public ScheduledInterviewRequest(
-      long id, TimeRange when, String interviewerId, String intervieweeId) {
+      long id, String dateString, String interviewer, String interviewee, String role) {
     this.id = id;
-    this.when = when;
-    this.interviewerId = interviewerId;
-    this.intervieweeId = intervieweeId;
+    this.dateString = dateString;
+    this.interviewer = interviewer;
+    this.interviewee = interviewee;
+    this.role = role;
```

Comment:
Q: l thought we were using role for interviewee/interviewer and position for SWE/PM/.... ?

Reply:
Not quite sure what the question is. The "role" here is what is displayed to the user who is participating in the interview. They are either an interviewer or interviewee depending on which id matches the user's id